tracer/src/Datadog.Tracer.Native/iast/dataflow_aspects.cpp

```diff
@@ -48,19 +48,32 @@ namespace iast
     }
 
     //------------------------------------
+    VersionInfo currentVersion = GetVersionInfo(GetDatadogVersion());
 
     DataflowAspectClass::DataflowAspectClass(Dataflow* dataflow, const WSTRING& aspectsAssembly, const WSTRING& line)
     {
         this->_dataflow = dataflow;
-        //[AspectClassAttribute("mscorlib,netstandard,System.Private.CoreLib",PROPAGATION,"")] Hdiv.AST.Aspects.Aspects.System_StringAspect
```

Comment:
nit: it's kind of handy having this example here when reading the logic below, so might be good to keep this + include an example of the "new" one? Same for below

Reply:
Agree